config.yml

```diff
@@ -50,6 +50,7 @@ params:
   current_version:      "4.6.0"
   current_ruby_version: "4.6.0"
   docs_version:         "4.6"
+  docs_version_short:   "4"
```

Comment:
as we follow bootstrap version, maybe use `current_version` to link to the tree, and just prefix with the v in footer link?

Reply:
The tree on Bootstrap repository is `v4-dev` and not `v4.6.0-dev`

Reply:
https://github.com/twbs/bootstrap/tree/v4.6.0 it's also tree for tags that are independent from branch that could be renamed